.github/workflows/build_and_test.yml

```diff
@@ -273,29 +273,27 @@ jobs:
           - ./fuzzers/binary-only/frida_executable_libpng
           - ./fuzzers/binary-only/frida_gdiplus
           - ./fuzzers/binary-only/frida_libpng
+          - ./fuzzers/binary-only/fuzzbench_qemu
           - ./fuzzers/binary-only/tinyinst_simple
 
           # Forkserver
           - ./fuzzers/forkserver/forkserver_simple
           - ./fuzzers/forkserver/forkserver_libafl_cc
+          - ./fuzzers/forkserver/fuzzbench_fork_qemu
+          - ./fuzzers/forkserver/fuzzbench_forkserver
+          - ./fuzzers/forkserver/fuzzbench_forkserver_cmplog
           - ./fuzzers/forkserver/libafl-fuzz
 
           # Full-system
```

Comment:
What about the qemu full system example?

Reply:
they should all be in Full-system. someone is missing?

Reply:
ah they are a bit below, to run with the qemu-compatible runners